pkgs/development/tools/build-managers/gradle/default.nix

```
@@ -128,10 +128,10 @@ rec {
   # https://docs.gradle.org/current/userguide/compatibility.html
 
   gradle_8 = gen {
-    version = "8.4";
+    version = "8.5";
     nativeVersion = "0.22-milestone-25";
-    sha256 = "1bkjxw7i0lm17pdyyvka4xpl6z0cdj0izagphync6839i2pg66iy";
-    defaultJava = jdk17;
+    sha256 = "09hhlka9vc4xr8xa68d8qcvyhsaaicrmh210x0wif23a0s3ng4lx";
```

```suggestion
    hash = "09hhlka9vc4xr8xa68d8qcvyhsaaicrmh210x0wif23a0s3ng4lx";
```

you will also need to change hash to SRI representation, and update param to `gen` function and inherit inside `gen`

---

Changed. The update script _should_ also be changed, but it uses `nix-prefetch-url`, which doesn’t output SRI format (yet?).

---

if this is too many changes, feel free to do it part of different PR, also If you wish I am happy to make that PR